perses/tests/test_repex.py

```diff
@@ -54,16 +54,15 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
             # Set up repex simulation
             reporter_file = os.path.join(temp_dir, f"{wt_name}-{mutant_name}.nc")
             reporter = MultiStateReporter(reporter_file, checkpoint_interval=10)
-            hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinSplittingDynamicsMove(timestep=4.0 * unit.femtoseconds,
-                                                                                   collision_rate=1.0 / unit.picosecond,
-                                                                                   n_steps=50,
-                                                                                   reassign_velocities=False,
-                                                                                   n_restart_attempts=20,
-                                                                                   splitting="V R R R O R R R V",
-                                                                                   constraint_tolerance=1e-06),
-                                                                                   replica_mixing_scheme='swap-all',
-                                    hybrid_factory=htf,
-                                    online_analysis_interval=None)
+            hss = HybridRepexSampler(mcmc_moves=mcmc.LangevinDynamicsMove(timestep=4.0 * unit.femtoseconds,
+                                                                          collision_rate=1.0 / unit.picosecond,
+                                                                          n_steps=50,
+                                                                          reassign_velocities=False,
+                                                                          n_restart_attempts=20,
+                                                                          constraint_tolerance=1e-06),
+                                     replica_mixing_scheme='swap-all',
+                                     hybrid_factory=htf,
+                                     online_analysis_interval=None)
             hss.setup(n_states=12, temperature=300 * unit.kelvin, t_max=300 * unit.kelvin,
                       storage_file=reporter, minimisation_steps=0, endstates=True)
```

Comment:
I think we _may_ need to minimize a bit here, since things could get pathologically slow if there are NaNs.

Reply:
`minimisation_steps=0` just means that minimization will run as many steps as needed until the energy is below the specified tolerance

Reply:
This seems like incorrect behavior: 
* `None` should be interpreted to run until tolerance is reached
* `0` should be interpreted to run 0 steps

Reply:
I think the reason why `minimisation_steps=0` means that it will run as many steps as needed is because this matches the default openmm behavior: http://docs.openmm.org/latest/api-python/generated/openmm.openmm.LocalEnergyMinimizer.html#openmm.openmm.LocalEnergyMinimizer.minimize

Maybe we could change `minimisation_steps` to `max_iterations` so that the matching behavior is clearer.

Reply:
`max_iterations=0` would still imply that the maximum number of iterations should be zero (no minimization).

The OpenMM choice was a poor default---it should have been -1 or some other choice.
Python has `None`, which should be the appropriate "please use the default" choice. 0 should mean 0 steps.

Reply:
Ok. I'll open an issue about this

Reply:
The issue is #1086 I'm considering this as "resolved". Feel free to re-open otherwise.

Reply:
@ijpulidos I think #1086 is a different issue and it hasn't been resolved yet. Can you clarify what you mean?